OpenRA.Mods.Common/Activities/Enter.cs

```diff
@@ -1,6 +1,6 @@
 #region Copyright & License Information
 /*
- * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
```

Comment:
Most of the changes in this file appear to be reverting #12601.  The downstream code needs to be updated instead.